docs/os/windows/windows-vm41.md

```diff
@@ -0,0 +1,203 @@
+
+- Importing from R4.0 should work, see [Migrate backups of Windows VMs created under Qubes R4.0 to R4.1](https://www.qubes-os.org/doc/windows-migrate41/).
+
+
+Windows VM installation
+-----------------------
+
+### qvm-create-windows-qube ###
+
+An unofficial, third-party tool for automating this process is available [here](https://github.com/elliotkillick/qvm-create-windows-qube).
+(Please note that this tool has not been reviewed by the Qubes OS Project.
+Use it at your own risk.)
+However, if you are an expert or want to do it manually you may continue below.
+
+### Summary ###
+
+~~~
+qvm-create --class StandaloneVM --label orange --property virt_mode=hvm win7new
+qvm-prefs WindowsNew memory 4096
+qvm-prefs WindowsNew maxmem 4096
+qvm-prefs WindowsNew kernel ''
+qvm-volume extend WindowsNew:root 50g
+qvm-start --cdrom=untrusted:/home/user/windows_install.iso WindowsNew
+# restart after the first part of the windows installation process ends
+qvm-start WindowsNew
+# once Windows is installed and working
+qvm-prefs WindowsNew qrexec_timeout 300
+~~~
+
+To install Qubes Windows Tools, follow instructions in [Qubes Windows Tools](https://www.qubes-os.org/doc/windows-tools41/).
+
+### Detailed instructions ###
+
+> **Notes:**
+> - The instructions may work on other versions than Windows 7, 10 and 11 x64 but haven't been tested.
+>- Qubes Windows Tools (QWT) only supports Windows 7, 10 and 11 x64. For installation, see [Qubes Windows Tools](https://www.qubes-os.org/doc/windows-tools41/).
+
+Create a VM named WindowsNew in [HVM](https://www.qubes-os.org/doc/hvm/) mode (Xen's current PVH limitations precludes from using PVH):
+
+~~~
+qvm-create --class StandaloneVM --label red --property virt_mode=hvm WindowsNew
+~~~
+
+Windows' installer requires a significant amount of memory or else the VM will crash with such errors:
+
+`/var/log/xen/console/hypervisor.log`:
+
+~~~
+p2m_pod_demand_populate: Dom120 out of PoD memory! (tot=102411 ents=921600 dom120)
+(XEN) domain_crash called from p2m-pod.c:1218
+(XEN) Domain 120 (vcpu#0) crashed on cpu#3:
+~~~
+
+So, increase the VM's memory to 4096MB (memory = maxmem because we don't use memory balancing).
+
+~~~
+qvm-prefs WindowsNew memory 4096
+qvm-prefs WindowsNew maxmem 4096
+~~~
+
+Disable direct boot so that the VM will go through the standard cdrom/HDD boot sequence:
+
+~~~
+qvm-prefs WindowsNew kernel ''
+~~~
+
+A typical Windows 7 installation requires between 25GB up to 60GB of disk space depending on the version (Home/Professional/...). Windows updates also end up using significant space. So, extend the root volume from the default 10GB to at least 50GB (note: it is straightforward to increase the root volume size after Windows is installed: simply extend the volume again in dom0 and then extend the system partition with Windows's disk manager).
+
+~~~
+qvm-volume extend WindowsNew:root 25g
```

Comment:
this should be 50g to be consistent with the summary and gui instructions ?

Reply:
Having burnt my fingers enough, I changed it everywhere to 60 GB. (I hope I found all locations.)